.rubocop.yml

```diff
@@ -32,6 +32,11 @@ Rails/SkipsModelValidations:
     - 'db/migrate_asset_refactor/*.rb'
     - 'features/support/step_definitions/*.rb'
 
```

Comment:
I was having some issues with the .rubocop.yml files in sub-directories, so I've pulled the configuration into here.